catalyst_voices/packages/internal/catalyst_voices_blocs/lib/src/session/session_state.dart

```diff
@@ -5,6 +5,13 @@ import 'package:flutter/material.dart';
 /// Determines the state of the user session.
 sealed class SessionState extends Equatable {
   const SessionState();
+
+  /// Returns a list of all available spaces
+  /// corresponding to the current session state.
+  List<Space> get spaces;
+
+  /// Returns a list of [spaces] that should be shown in overall spaces menu.
+  List<Space> get overallSpaces;
 }
```

Comment:
Not but, but in future, i would love to `SessionState` to be just **final class** instead of **sealed**